WikiWatershed/model-my-watershed
# BiG-CZ: List Popup

## Overview

When BiG-CZ shapes overlapped we were only able to display the topmost layer. Add a popup that allows browsing details of all the shapes at the clicked point.

Connects #2282 

### Demo

Multipage

![h1d0kybyd2](https://user-images.githubusercontent.com/7633670/31036738-a19dcd88-a53b-11e7-9742-b273c9f76991.gif)

Single page

![screen shot 2017-09-29 at 5 03 44 pm](https://user-images.githubusercontent.com/7633670/31036763-b1f79f06-a53b-11e7-9bb9-e60fb0eb26a0.png)

### Notes


![gk1zolrtuu](https://user-images.githubusercontent.com/7633670/31036745-a773e76a-a53b-11e7-863f-f4a78bdba58b.gif)

I left the height dynamic for now. When there's only a single item on a page, the above height "jumping" happens. I'm not sure what's desirable @jfrankl 

## Testing Instructions

 * Pull this branch and `bundle`
 * Spend some time searching and clicking the results of various catalogs
     - If there's a single shape under your click, the details popup should show
     - If multiple, the list popup should show
     - You should still be able to highlight features by hovering over the sidebar items
     - Searching, switching catalogs, and opening the details of a result should close the popup
